examples/restclient-pact/src/main/kotlin/cnt/adapters/restclient/model/Actor.kt

```diff
@@ -0,0 +1,14 @@
+package cnt.adapters.restclient.model
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.quarkus.runtime.annotations.RegisterForReflection
+
+@RegisterForReflection
+data class Actor(
+    @field:JsonProperty("id")
```

Comment:
Question for my understanding:
The `JsonProperty` annotations are needed because there is no at-runtime reflection in Quarkus. I thought that `@RegisterForReflection` would take care of the field names. Is it really necessary for Jackson to work in a Quarkus app to annotate each and every field manually?

Reply:
with quarkus 2.16. not needed anymore